packages/fxa-content-server/server/lib/validation.js

```diff
@@ -49,6 +49,7 @@ const TYPES = {
     .uri({ scheme: ['android-app', 'http', 'https'] })
     .allow('none'),
   RESUME: joi.string().regex(PATTERNS.BASE64),
+  SETTINGS_VERSION: joi.string().valid('old', 'new'),
```

Comment:
I'm not 100% sold on the values here but don't have any suggestions either 😅

Reply:
These were suggested by @irrationalagent on Slack.